UnityGLTF/Assets/UnityGLTF/Scripts/GLTFSceneImporter.cs

```diff
@@ -664,6 +710,7 @@ protected virtual async Task ConstructUnityTexture(Stream stream, bool markGpuOn
 				if (_asyncCoroutineHelper != null) await _asyncCoroutineHelper.YieldOnTimeout();
 				//	NOTE: the second parameter of LoadImage() marks non-readable, but we can't mark it until after we call Apply()
 				texture.LoadImage(buffer, markGpuOnly);
+				Progress.TextureLoaded++;
```

Comment:
Instead of two increments in the if/else block in this function, you could have a single increment at the end of the if/else blocks.